matrixprofile/analyze.py

```diff
@@ -218,6 +225,15 @@ def analyze(ts, query=None, windows=None, sample_pct=1.0, threshold=0.98, n_jobs
     """
     result = None
 
+    # preprocess the time series
+    preprocessing_kwargs = validate_preprocess_kwargs(preprocessing_kwargs)
+    if preprocessing_kwargs != None:
```

Comment:
This can be simplified to:

```
if preprocessing_kwargs:
```

In Python, False and None do not validate the condition.